### ISSUE TITLE
Clicking Timestamp doesnt jump to time in the video

### ISSUE THREAD

**Issue by mishipal** [aidenlx/media-extended#434]
Clicking on Timestamp doesnt jump the video to desired time but rather asks to open link externally. Please help.
![image](https://github.com/user-attachments/assets/09550431-1aed-4e82-a024-cd7ec52764cb)

![image](https://github.com/user-attachments/assets/6f70c650-308c-4e59-899a-bdadcba492b8)

**Comment by Mohammed159159** [comment]
Is this on the latest version? Suddenly, since 2.11.1, this plugin stopped working. So, I am interested to know if it works for you on a later version.

**Comment by mishipal** [comment]
I am using Media Extended 3.2.6 with Obsidian 1.7.5. The app is very unstable and I am no longer able to trust it to work.

**Comment by Mohammed159159** [comment]
> I am using Media Extended 3.2.6 with Obsidian 1.7.5. The app is very unstable and I am no longer able to trust it to work.

You can use version 2.11.1. It is working properly but misses some features from the later versions, most importantly the screenshot feature.

**Comment by juestchaos** [comment]
@mishipal  You can provide your Obsidian version and plugin version number. Alternatively, you can provide whether it is a local video or an online video URL, so we can help to locate the issue.

**Comment by Medullitus** [comment]
Same issue. Online video. Obsidian version is 1.7.7 Appimage. Extended version is 3.2.6



> @mishipal You can provide your Obsidian version and plugin version number. Alternatively, you can provide whether it is a local video or an online video URL, so we can help to locate the issue.

**Comment by Medullitus** [comment]
> > I am using Media Extended 3.2.6 with Obsidian 1.7.5. The app is very unstable and I am no longer able to trust it to work.
> 
> You can use version 2.11.1. It is working properly but misses some features from the later versions, most importantly the screenshot feature.

The latest version is quite stable. There's no need to use the version from 4 years ago, my brother Muslim.

**Comment by Mohammed159159** [comment]
> > > I am using Media Extended 3.2.6 with Obsidian 1.7.5. The app is very unstable and I am no longer able to trust it to work.
> > 
> > 
> > You can use version 2.11.1. It is working properly but misses some features from the later versions, most importantly the screenshot feature.
> 
> The latest version is quite stable. There's no need to use the version from 4 years ago, my brother Muslim.

Yes. People discovered here #438 that it works, even with online videos, if you open them as `iframe` instead of webpage, but you lose the screenshot feature. You only need to confirm it's actually opened as `iframe`. See the issue for more details. You're probably having fun with YT premium anyways 😅.
